contracts/DecentralizedAutonomousTrust.sol

```diff
@@ -880,16 +881,16 @@ contract DecentralizedAutonomousTrust is ERC20, ERC20Detailed {
     require(sent, 'Failed to send Ether');
   }
 
-  function mint(address to, uint256 amount) public onlyControl {
-     super._mint(to, amount);
-  }
-
   /**
-   * Burns tokens from account
-   * @param account account address
-   * @param amount amount to mint
+   * Saves vested token in vesting contract to the vesting beneficiary
+   * Used to accelerate vesting
+   * @param tokenVesting token vesting contract
    */
-  function burnFrom(address account, uint256 amount) public onlyControl {
-    _burn(account, amount, false);
+  function saveVestedTokens(ITokenVesting tokenVesting) public onlyControl {
+    uint256 tokenVestingBalance = balanceOf(address(tokenVesting));
+    require(tokenVestingBalance > 0, "DAT: No vested tokens to save");
+    // Burn from the vesting contract and mint to the beneficiary
+    super._burn(address(tokenVesting), tokenVestingBalance);
```

Comment:
why not use _transfer here instead of minting and burning

Reply:
Addressed in [51ef5c8](https://github.com/3acvc/dxd-token/pull/3/commits/51ef5c8c70012b90b83b48adf8b59e871075b0cc)